template/template/clash.tpl

```diff
@@ -50,7 +65,7 @@ Rule:
 {{ youtube_rules.main('🚀 Proxy') | clash }}
 {{ us_rules.main('US') | clash }}
 {{ remoteSnippets.telegram.main('🚀 Proxy') | clash }}
-{{ blocked_rules.main('🚀 Proxy') | clash }}
```

Comment:
You got it wrong, it means "blocked by the GFW".

Reply:
:rofl: oh god my bad. I was thinking of ad-blocking. Reverted.